src/app/project/ProjectContainer.js

```diff
@@ -121,21 +116,22 @@ define([
             // response
             console.log('app.project.ProjectContainer::_updateDetails', arguments);
 
-            if (this.projectDetails) {
-                this.projectDetails.destroy();
-                this.featuresGrid.destroy();
-
-                this.projectDetails = null;
-                this.featuresGrid = null;
+            if (this.featureDetails) {
+                [this.featureDetails, this.projectDetails, this.featuresGrid].forEach(function (widget) {
+                    widget.destroy();
+                    widget = null;
```

Comment:
Not sure this is needed but couldn't hurt I guess. Since `widget` is scoped just to this anonymous function then it's going to be garbage collected anyway when the function is finished, right?